src/iop/graduatednd.c

```diff
@@ -43,6 +43,10 @@
 
 #if defined(__SSE__)
```

Comment:
Since SSE is removed here, can't we remove this part?

Reply:
Although the SSE codepath has been removed, we still need that one instruction to maintain overall performance.  I can integrate it as part of the copy_pixel_nontemporal definition in darktable.h, which I've since noticed does in fact include the header file.

Reply:
But if one compiles without SSE support it won't be defined. So if call outside of an SSE section a missing fct will be reported, no?

Reply:
No, because darktable.h defines an inline function - but I haven't seen the version from darktable.h actually generating a nontemporal move instruction.